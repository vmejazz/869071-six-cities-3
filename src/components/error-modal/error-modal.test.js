import React from "react";
import renderer from "react-test-renderer";
import ErrorModal from "./error-modal.jsx";

it(`<ErrorModal /> should render `, () => {
  const tree = renderer
      .create(
          <ErrorModal
            closeModal={() => {}}
          />
      )
      .toJSON();

  expect(tree).toMatchSnapshot();
});
