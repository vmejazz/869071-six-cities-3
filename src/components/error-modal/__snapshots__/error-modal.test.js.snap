// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ErrorModal /> should render  1`] = `
<div
  onClick={[Function]}
  style={
    Object {
      "background": "rgba(0,0,0,0.8)",
      "bottom": "0",
      "left": "0",
      "position": "fixed",
      "right": "0",
      "top": "0",
      "transition": "opacity 400ms ease-in",
      "zIndex": "99999",
    }
  }
>
  <p
    style={
      Object {
        "background": "#fff",
        "borderRadius": "10px",
        "display": "block",
        "fontSize": "28px",
        "height": "100px",
        "margin": "10% auto",
        "padding": "5px 20px 13px 20px",
        "paddingTop": "50px",
        "position": "relative",
        "textAlign": "center",
        "width": "400px",
      }
    }
  >
    Возникла сетевая ошибка
  </p>
</div>
`;
